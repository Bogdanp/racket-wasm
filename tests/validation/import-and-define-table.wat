(module
 (import "a" "t" (table 1 funcref))
 (table 1 funcref))