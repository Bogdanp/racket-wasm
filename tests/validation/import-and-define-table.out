(#f . "only one table allowed\n  at: (tabletype '(elemtype funcref) (limits 1 #f))")